packages/core-web/src/styles/markbind.css

```diff
@@ -118,11 +118,19 @@ code.hljs.inline {
 /* Header */
 
 header[fixed] {
+    max-height: 100%;
+    overflow: hidden;
```

Comment:
will need to be dynamically adjusted (`visible` once it reaches max-height 100%, `hidden` otherwise) 

searchbar:
![Untitled](https://user-images.githubusercontent.com/3306138/107840564-bdddb600-6dee-11eb-9716-a562e35623d8.png)

Reply:
I've managed to fixed this by moving the `overflow: hidden` to a media query for mobile. Seems to be working for both mobile and desktop sites. Is this approach ok instead of looking at the max-height?

Reply:
hmm I guessing it may not work well if the user uses a popover, etc. in the header 

also there's the upper topnav of items in #980 and how that might work with dropdowns (might overflow similarly)

Reply:
Alright got it 👍. Updated `overflow` to be dynamic such that it will only be visible if `max-height` is 100%.